src/components/WithinContentContext/context.tsx

```diff
@@ -1,11 +1,9 @@
 import React from 'react';
 
-export interface WithinContentContextType {
-  withinContentContainer: boolean;
-}
+export type WithinContentContextType = boolean;
```

Comment:
Having to define and export such a trivial type just so we can use it in tests feels a bit rubbish.

What do you think to removing this and updating the tests so the TestComponent is `const TestComponent = (_: any) => null;`?